src/encoded/types/workflow.py

```diff
@@ -1,34 +1,23 @@
 """The type file for the workflow related items.
 """
-from collections import (
-    OrderedDict,
-    deque
-)
-from inspect import signature
+
 import copy
-from pyramid.view import view_config
-from pyramid.response import Response
-from pyramid.httpexceptions import (
-    HTTPUnprocessableEntity,
-)
-from snovault import (
-    calculated_property,
-    collection,
-    load_schema,
-    CONNECTION,
-    TYPES
-)
-from snovault.util import debug_log
-from .base import (
-    Item,
-    lab_award_attribution_embed_list
-)
 import cProfile
-import pstats
-import io
 import boto3
+import io
 import json
+import pstats
+
+from collections import OrderedDict, deque
+from dcicutils.env_utils import FF_ENV_WEBDEV, is_stg_or_prd_env, prod_bucket_env
+from inspect import signature
+from pyramid.httpexceptions import HTTPUnprocessableEntity
+from pyramid.response import Response
+from pyramid.view import view_config
+from snovault import calculated_property, collection, load_schema, CONNECTION, TYPES
+from snovault.util import debug_log
 from time import sleep
+from .base import Item, lab_award_attribution_embed_list
```

Comment:
Mostly these imports are just rearranged (alphabetized) to make it easier to see what's there. The only difference of substance is the introduction of the `dcicutils.env_utils` line.